kore/test/Test/Kore/OnePath/Step.hs

```diff
@@ -312,19 +312,9 @@ test_onePathStrategy =
                     $ Mock.f Mock.b
             ]
         assertEqualWithExplanation ""
-            [ Stuck Conditional
```

Comment:
This change does not seem reasonable to me. We cannot unconditionally prove the claim `constr10(b) => a` given only the axiom `constr10(b) => a requires f(b) = c`.